README.md

```diff
@@ -0,0 +1,76 @@
+# Gitlab-Slack Notifier
+
+This project has the objective to integrate Gitlab with Slack teams through webhooks. We create a service in NodeJS, that handle the hooks and send messages to channels and direct messages to slack users. 
+
+So far, we integrated *Push Hooks* and *Merge Request Hooks*.
+
+**IMPORTANT:** If you want to notify the slack users through DM, they must have the same username in Gitlab and Slack.
+##### Ex: Slack username: @brunojdo and Gitlab username: brunojdo
+
+### Specs
+
+* NodeJS 8.3
+* Express 4.10.*
+
+## Usage
+
+### Docker
+
+You can start the service using container. Execute the command below: 
+
+`docker run -d --restart=always -p 8008:8080 --name notifier-slgb brunodias20/slack-gitlab-notifier:{version}`
+
+### NPM 
+
+If you prefer, you also may clone this project, and into `app/` folder, execute: 
```

Comment:
If you prefer, you may also clone this project and run `npm`. On the `./app` folder you can execute: